docker-compose.yml

```diff
@@ -1,23 +1,21 @@
-version: '3.0'
-services:
+version: '3.8'
 
+services:
   elasticsearch:
     image: amsterdam/elasticsearch6
-    command:  elasticsearch -Ehttp.host=0.0.0.0 -Etransport.host=127.0.0.1
-    volumes:
-      - "~/.ssh/datapunt.key:/root/.ssh/datapunt.key"
+    command: elasticsearch -Ehttp.host=0.0.0.0 -Etransport.host=localhost
     ports:
-      - "9200:9200"
-      - "9300:9300"
+      - '9200:9200'
+      - '9300:9300'
 
   cms_search:
     build: .
     ports:
-      - "8080:8080"
+      - '8080:8080'
     volumes:
-      - $PWD/data:/app/data
+      - ./data:/app/data
     environment:
       - DATAPUNT_API_URL=${DATAPUNT_API_URL:-https://api.data.amsterdam.nl/}
     depends_on:
       - elasticsearch
-
+    env_file: ./.env.development
```

Comment:
Why are we always including the development environment?

Reply:
Locally, we need to. Otherwise, the required env vars won't be available in the application.